conda_build/environ.py

```diff
@@ -144,6 +144,7 @@ def get_dict(m=None, prefix=None):
         d['PKG_NAME'] = m.name()
         d['PKG_VERSION'] = m.version()
         d['PKG_BUILDNUM'] = str(m.build_number())
+        d['PKG_BUILD_STRING'] = str(m.build_id())
```

Comment:
No, I meant:

```
d['recipe'] = m
```

Within Jinja, we should be able to do things like:

```
{{ recipe.name() }} 
```

right?